examples/plot_bootstrapped_coffecients_for_lm.py

```diff
@@ -0,0 +1,144 @@
+"""
+================================================================
+Plot bootstrapped beta coefficients for a linear model estimator
+================================================================
+
+"""
+
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from sklearn.linear_model import LinearRegression
+
+from mne.decoding import Vectorizer, get_coef
+from mne.datasets import limo
+from mne.evoked import EvokedArray
+from mne.io.pick import pick_types
+
+###############################################################################
+# list with subjects ids that should be imported
+subjects = [2]
+# create a dictionary containing participants data
+limo_epochs = {str(subj): limo.load_data(subject=subj) for subj in subjects}
+
+# get key from subjects dict for easy slicing
+# subjects = list(limo_epochs.keys())
+
+# interpolate missing channels
+for subject in limo_epochs.values():
+    subject.interpolate_bads(reset_bads=True)
+
+# pick channels that should be included in the analysis
+picks_eeg = pick_types(limo_epochs['2'].info, eeg=True)
+# channels to be excluded
+exclude = ['EXG1', 'EXG2', 'EXG3', 'EXG4']
+
+# save epochs information (needed for creating a homologous
+# epochs object containing linear regression result)
+epochs_info = limo_epochs['2'].copy().drop_channels(exclude).info
+tmin = limo_epochs['2'].tmin.copy()
```

Comment:
Why copy an int?

Reply:
this is just me being too anxious about shallow copies :grimacing: